hugolib/page.go

```diff
@@ -52,7 +52,7 @@ const (
 	// The rest are node types; home page, sections etc.
 	KindHome         = "home"
 	KindSection      = "section"
-	KindTaxonomy     = "taxonomy"
+	KindTaxonomy     = "taxtonomy"
```

Comment:
It's not my kind of music, I guess it is country/folk rock, whatever ... But by glimpsing at your name, I saw those letters. Never mind.